### PR TITLE
More validation tweaks

### PR DESCRIPTION
**Pull request by mverzett**
* Disable plotting for branches that perfectly match when validating 
   * Plot added branches, without comparison
   * Refactor timing checks, make a table comparing the different modules
   * use [inspectNanoFile.py](https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/test/inspectNanoFile.py) to make html pages of the file size (requires checking out `PhysicsTools/NanoAOD`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ mv $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoEgamma/ElectronIdentification/data/
 ## Add the BParkingNano package and build everything
 
 ```
+git cms-addpkg PhysicsTools/NanoAOD
```
Comment (amartelli) [inline README.md]: why NanoAOD needed explicitly?
Also, since you are modifying the README can I ask to update the instructions  "To run on a test file"

Comment (amartelli) [reply README.md]: sorry just read with more attention your comments :) NanoAOD is needed explicitly!

**Comment by mverzett** [comment]
@sarafiorendi @amartelli are you OK with this PR? Can we go ahead and merge?

**Comment by amartelli** [comment]
yes just the README that requires a general update, but it's not important really (at least now)

I am fine with this

**Pull request merged**